src/mibase.ts

```diff
@@ -84,9 +84,9 @@ export class MI2DebugSession extends DebugSession {
 				if (process.platform != "win32")
 					this.handleMsg("stderr", "Code-Debug WARNING: Utility Command Server: Error in command socket " + err.toString() + "\nCode-Debug WARNING: The examine memory location command won't work");
 			});
-			if (!fs.existsSync(systemPath.join(os.tmpdir(), "code-debug-sockets")))
-				fs.mkdirSync(systemPath.join(os.tmpdir(), "code-debug-sockets"));
-			this.commandServer.listen(this.serverPath = systemPath.join(os.tmpdir(), "code-debug-sockets", ("Debug-Instance-" + Math.floor(Math.random() * 36 * 36 * 36 * 36).toString(36)).toLowerCase()));
+			if (!existsSync(join(tmpdir(), "code-debug-sockets")))
```

Comment:
keep `systemPath`, it's written explicitly to differentiate that this is `path` using separators using the host OS config, rather than the debuggerPath, which uses separates on the platform that the debugger is running on